blocked-edges/4.12.12-PatchesOlderRelease.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.12.12
+from: 4[.](11[.]35|12[.]11)
+url: https://access.redhat.com/solutions/7007136
+name: PatchesOlderRelease
+message: |-
+  4.12.12 patches 4.12.10 to fix https://issues.redhat.com//browse/OCPBUGS-11662 , which is an installer issue, and not and issue with running clusters.  A number of other changes had landed since 4.12.10 and were shipped in 4.12.11.  We expect those fixes to be useful, but they might also introduce regressions, and because they have not soaked for as long, we did not include them in 4.12.12.  But that means we know 4.12.12 has regressed on all of those bugs compared to 4.12.10, and possibly also its weekly sibling 4.11.25, so we do not recommend updating 4.11.35 or 4.12.11 clusters to 4.12.12.  Later 4.12.z will include both 4.12.12's installer fix and 4.12.11's fixes, and be recommended update targets for 4.11.35 and 4.12.11.
```

Comment:
Feels like we should be more brief and "strict"? Not sure whether readers really appreciate the gory details (or if we really need them to know).

I'd go with this:

The 4.12.12 release resolves https://issues.redhat.com//browse/OCPBUGS-11662, which is an installation issue and does not affect already running clusters. 4.12.12 does not include fixes from 4.12.11 and therefore upgrading from 4.12.11 would regress them. Red Hat does not recommend upgrading 4.11.35 or 4.12.11 clusters to 4.12.12 for this reason.